go.mod

```diff
@@ -4,13 +4,12 @@ go 1.16
 
 require (
 	github.com/google/go-querystring v1.1.0 // indirect
-	github.com/hashicorp/go-cleanhttp v0.5.2 // indirect
 	github.com/hashicorp/go-retryablehttp v0.6.8 // indirect
 	github.com/hashicorp/go-slug v0.7.0 // indirect
 	github.com/hashicorp/go-tfe v0.14.0
 	github.com/hashicorp/go-version v1.2.1
 	github.com/hashicorp/hcl v0.0.0-20180404174102-ef8a98b0bbce
-	github.com/hashicorp/terraform-plugin-sdk/v2 v2.3.0
+	github.com/hashicorp/terraform-plugin-sdk/v2 v2.6.0
```

Comment:
? Did you intend for an SDK bump in these changes?

Reply:
Yup, its to get the acceptance tests to work: https://github.com/hashicorp/terraform-plugin-sdk/blob/main/CHANGELOG.md#243-february-10-2021

Reply:
Ah excellent!